### PR TITLE
docs(operators): add documentation for windowCount

### PR DESCRIPTION
**Pull request by hardikpthv**
Closes: #125

### CODE REVIEW

File: src/operator-docs/transformation/windowCount.ts [hunk 1]
```diff
@@ -1,6 +1,91 @@
...
+          .map(win => win.skip(1)) // skip first of every 3 clicks
+          .mergeAll(); // flatten the Observable-of-Observables
+        result.subscribe(x => console.log(x));
+      `,
+      externalLink: {
+        platform: 'JSBin',
+        url: 'http://jsbin.com/haruca/embed?js,console,output'
+      }
+    },
+    {
```
Comment (JWO719) [inline src/operator-docs/transformation/windowCount.ts]: These examples are more or less the same. I think one can be removed.

Comment (hardikpthv) [reply src/operator-docs/transformation/windowCount.ts]: Alright, i will remove other one.

File: src/operator-docs/transformation/windowCount.ts [hunk 2]
```diff
@@ -1,6 +1,77 @@
...
+        startWindowEvery</span> is not provided, then new windows are started immediately at the start of the
+        source and when each window completes with size <span class="markdown-code">windowSize</span>.
+      </p>
+    `
+  },
+  examples: [
+    {
+      name: 'Ignore every 3rd click event, starting from the third one',
+      code: `
+        const clicks = Rx.Observable.fromEvent(document, 'click');
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/transformation/windowCount.ts]: Can you please update the examples to es6 imports and lettable operators ?

Comment (hardikpthv) [reply src/operator-docs/transformation/windowCount.ts]: Okay.

File: src/operator-docs/transformation/windowCount.ts [hunk 3]
```diff
@@ -1,6 +1,77 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowCount: OperatorDoc = {
-  'name': 'windowCount',
-  'operatorType': 'transformation'
+  name: 'windowCount',
+  operatorType: 'transformation',
+  signature:
+    'public windowCount(windowSize: number, startWindowEvery: number): Observable<Observable<T>>',
```
Comment (sumitarora) [inline src/operator-docs/transformation/windowCount.ts]: Please remove `<T>` we have decided not to use generics.

File: src/operator-docs/transformation/windowCount.ts [hunk 4]
```diff
@@ -1,6 +1,77 @@
...
+  },
+  examples: [
+    {
+      name: 'Ignore every 3rd click event, starting from the third one',
+      code: `
+        const clicks = Rx.Observable.fromEvent(document, 'click');
+        const result = clicks.windowCount(2, 3)
+          .mergeAll(); // flatten the Observable-of-Observables
+        result.subscribe(x => console.log(x));
+      `,
```
Comment (sumitarora) [inline src/operator-docs/transformation/windowCount.ts]: Could you please add expected output?

Comment (hardikpthv) [reply src/operator-docs/transformation/windowCount.ts]: Got it.

File: src/operator-docs/transformation/windowCount.ts (hunk 3 above)
Comment (hardikpthv) [reply src/operator-docs/transformation/windowCount.ts]: Yep...

**Comment by JWO719** [comment]
@sumitarora could you take a look at this one to get it into master?

**Pull request merged**